release-notes/7.0.0.md

```diff
@@ -0,0 +1,49 @@
+# Release Note 7.0.0 ML projects Service
+
+This version contains set of manual activites tasks that must be completed in order to improve to upgrade the ML Projects service code to 7.0.0. Please consider the following list of tasks to be completed.
+
+### Deploy ml-projects-services
+
+To retrieve the latest release tag for version 7.0.0, please visit the following URL: https://github.com/project-sunbird/ml-projects-service/tags e.g. release-7.0.0_RC1
+
+To proceed with the deployment process, follow the steps below:
+
+    1. Log in to Jenkins.
+    2. Navigate to Dashboard -> AutoBuild -> StagingManual -> managed-learn -> ml-projects-service. OR for dev Navigate to Dashboard -> Build -> managed-learn -> ml-projects-service
+    3. Click on "Build with parameters" and provide the latest release tag in the field labeled "github_release_tag". Initiate the build process.
+    4. The build job will take approximately 5 minutes to complete.
+    5. After the job finishes, go to Dashboard -> Deploy -> staging -> managed-learn -> ml-projects-service. OR for dev go to Dashboard -> Deploy -> dev -> managed-learn -> ml-projects-service This job will be executed automatically in the dev environment. If not, then it should be built manually.
+    6. Click on "Build with parameters" to initiate the deployment process.
+    7. Once the job is completed, the services will be deployed on the staging environment
+
+## Migrations
```

Comment:
@ankitpws - Mention that this release contains certain other migrations and link the readme for the same.

Reply:
removed as it will go in 6.0.0 release